This patch adds the blacklisting for all the management luns. Otherwise
user has to manually add blacklisting in multipath.conf for these luns.

Signed-off-by: Babu Moger <babu.moger@netapp.com>
---
---
 libmultipath/hwtable.c |   19 +++++++++++++++++++
 1 file changed, 19 insertions(+)

Index: multipath-tools/libmultipath/hwtable.c
===================================================================
--- multipath-tools.orig/libmultipath/hwtable.c
+++ multipath-tools/libmultipath/hwtable.c
@@ -449,6 +449,7 @@ static struct hwentry default_hw[] = {
 		/* IBM FAStT 1722-600 */
 		.vendor        = "IBM",
 		.product       = "^1722-600",
+		.bl_product    = "Universal Xport",
 		.getuid        = DEFAULT_GETUID,
 		.features      = "1 queue_if_no_path",
 		.hwhandler     = "1 rdac",
@@ -466,6 +467,7 @@ static struct hwentry default_hw[] = {
 		/* IBM DS4100 */
 		.vendor        = "IBM",
 		.product       = "^1724",
+		.bl_product    = "Universal Xport",
 		.getuid        = DEFAULT_GETUID,
 		.features      = "1 queue_if_no_path",
 		.hwhandler     = "1 rdac",
@@ -483,6 +485,7 @@ static struct hwentry default_hw[] = {
 		/* IBM DS3200 / DS3300 / DS3400 */
 		.vendor        = "IBM",
 		.product       = "^1726",
+		.bl_product    = "Universal Xport",
 		.getuid        = DEFAULT_GETUID,
 		.features      = "1 queue_if_no_path",
 		.hwhandler     = "1 rdac",
@@ -500,6 +503,7 @@ static struct hwentry default_hw[] = {
 		/* IBM DS4400 / DS4500 / FAStT700 */
 		.vendor        = "IBM",
 		.product       = "^1742",
+		.bl_product    = "Universal Xport",
 		.getuid        = DEFAULT_GETUID,
 		.features      = DEFAULT_FEATURES,
 		.hwhandler     = "1 rdac",
@@ -516,6 +520,7 @@ static struct hwentry default_hw[] = {
 	{
 		.vendor        = "IBM",
 		.product       = "^1745|^1746",
+		.bl_product    = "Universal Xport",
 		.getuid        = DEFAULT_GETUID,
 		.features      = "2 pg_init_retries 50",
 		.hwhandler     = "1 rdac",
@@ -565,6 +570,7 @@ static struct hwentry default_hw[] = {
 	    /* IBM DS4700 */
 		.vendor        = "IBM",
 		.product       = "^1814",
+		.bl_product    = "Universal Xport",
 		.getuid        = DEFAULT_GETUID,
 		.features      = DEFAULT_FEATURES,
 		.hwhandler     = "1 rdac",
@@ -582,6 +588,7 @@ static struct hwentry default_hw[] = {
 	    /* IBM DS4800 */
 		.vendor        = "IBM",
 		.product       = "^1815",
+		.bl_product    = "Universal Xport",
 		.getuid        = DEFAULT_GETUID,
 		.features      = DEFAULT_FEATURES,
 		.hwhandler     = "1 rdac",
@@ -599,6 +606,7 @@ static struct hwentry default_hw[] = {
 	    /* IBM DS5000 */
 		.vendor        = "IBM",
 		.product       = "^1818",
+		.bl_product    = "Universal Xport",
 		.getuid        = DEFAULT_GETUID,
 		.features      = DEFAULT_FEATURES,
 		.hwhandler     = "1 rdac",
@@ -616,6 +624,7 @@ static struct hwentry default_hw[] = {
 		/* IBM Netfinity Fibre Channel RAID Controller Unit */
 		.vendor        = "IBM",
 		.product       = "^3526",
+		.bl_product    = "Universal Xport",
 		.getuid        = DEFAULT_GETUID,
 		.features      = DEFAULT_FEATURES,
 		.hwhandler     = "1 rdac",
@@ -918,6 +927,7 @@ static struct hwentry default_hw[] = {
 		/* DELL MD3000 */
 		.vendor        = "DELL",
 		.product       = "MD3000",
+		.bl_product    = "Universal Xport",
 		.getuid        = DEFAULT_GETUID,
 		.features      = "2 pg_init_retries 50",
 		.hwhandler     = "1 rdac",
@@ -935,6 +945,7 @@ static struct hwentry default_hw[] = {
 		/* DELL MD3000i */
 		.vendor        = "DELL",
 		.product       = "MD3000i",
+		.bl_product    = "Universal Xport",
 		.getuid        = DEFAULT_GETUID,
 		.features      = "2 pg_init_retries 50",
 		.hwhandler     = "1 rdac",
@@ -952,6 +963,7 @@ static struct hwentry default_hw[] = {
 		/* DELL MD32xx */
 		.vendor        = "DELL",
 		.product       = "MD32xx",
+		.bl_product    = "Universal Xport",
 		.getuid        = DEFAULT_GETUID,
 		.features      = "2 pg_init_retries 50",
 		.hwhandler     = "1 rdac",
@@ -969,6 +981,7 @@ static struct hwentry default_hw[] = {
 		/* DELL MD32xxi */
 		.vendor        = "DELL",
 		.product       = "MD32xxi",
+		.bl_product    = "Universal Xport",
 		.getuid        = DEFAULT_GETUID,
 		.features      = "2 pg_init_retries 50",
 		.hwhandler     = "1 rdac",
@@ -1076,6 +1089,7 @@ static struct hwentry default_hw[] = {
 	{
 		.vendor        = "SGI",
 		.product       = "TP9[45]00",
+		.bl_product    = "Universal Xport",
 		.getuid        = DEFAULT_GETUID,
 		.features      = DEFAULT_FEATURES,
 		.hwhandler     = "1 rdac",
@@ -1092,6 +1106,7 @@ static struct hwentry default_hw[] = {
 	{
 		.vendor        = "SGI",
 		.product       = "IS.*",
+		.bl_product    = "Universal Xport",
 		.getuid        = DEFAULT_GETUID,
 		.features      = "2 pg_init_retries 50",
 		.hwhandler     = "1 rdac",
@@ -1114,6 +1129,7 @@ static struct hwentry default_hw[] = {
 	{
 		.vendor        = "STK",
 		.product       = "OPENstorage D280",
+		.bl_product    = "Universal Xport",
 		.getuid        = DEFAULT_GETUID,
 		.features      = DEFAULT_FEATURES,
 		.hwhandler     = "1 rdac",
@@ -1174,6 +1190,7 @@ static struct hwentry default_hw[] = {
 	{
 		.vendor        = "SUN",
 		.product       = "CSM200_R",
+		.bl_product    = "Universal Xport",
 		.getuid        = DEFAULT_GETUID,
 		.features      = DEFAULT_FEATURES,
 		.hwhandler     = "1 rdac",
@@ -1191,6 +1208,7 @@ static struct hwentry default_hw[] = {
 	{
 		.vendor        = "SUN",
 		.product       = "LCSM100_[IEFS]",
+		.bl_product    = "Universal Xport",
 		.getuid        = DEFAULT_GETUID,
 		.features      = DEFAULT_FEATURES,
 		.hwhandler     = "1 rdac",
@@ -1208,6 +1226,7 @@ static struct hwentry default_hw[] = {
 	{
 		.vendor        = "(LSI|ENGENIO)",
 		.product       = "INF-01-00",
+		.bl_product    = "Universal Xport",
 		.getuid        = DEFAULT_GETUID,
 		.features      = "2 pg_init_retries 50",
 		.hwhandler     = "1 rdac",
